cmd/root.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "kube-compose",
+	Short: "k8s",
+	Long:  `Environments on k8s made easy`,
+}
+
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	rootCmd.PersistentFlags().StringP("namespace", "n", "", "namespace for environment (required)")
+	rootCmd.MarkPersistentFlagRequired("namespace")
```

Comment:
Namespace should not be required, before refactoring, if neither the namespace flag or environment variable is set, it will take the namespace of the current context in the kube compose file.

Reply:
Removed.